users/views.py

```diff
@@ -11,7 +11,10 @@
 from rest_framework.authtoken.models import Token
 from rest_framework.permissions import IsAuthenticated
 from rest_framework.decorators import api_view, permission_classes
-from org.models import *
```

Comment:
Bro please add `from org.models import *` , the ListOrgsTestCase is failing because maybe you accidentally deleted this line.

Reply:
okay sir